src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+
+import { Avatar as CoreAvatar } from 'wix-ui-core/avatar';
+
+import styles from './Avatar.st.css';
+
+export enum AvatarSize {
+  xLarge = 'xLarge',
+  large = 'large',
+  medium = 'medium',
+  small = 'small',
+}
+
+export interface AvatarProps {
+  name?: string;
+  size?: AvatarSize;
+  src?: string;
+}
+
+interface DefaultProps {
+  size: AvatarSize;
+}
+
+/** Avatar is a type of element that visually represents a user, either as an image, placeholder or text (name initials). */
+export class Avatar extends React.Component<AvatarProps> {
+  static displayName = 'Avatar';
+  static defaultProps: DefaultProps = {
+    size: 'medium' as AvatarSize,
+  };
+
+  render() {
+    const { size, src, name, ...rest } = this.props;
+
+    const placeholder = getDefaultAvatar(size);
+
+    return (
+      <CoreAvatar
+        {...rest}
+        {...styles('root', { size }, rest)}
+        imgProps={{ src }}
+        placeholder={placeholder()}
+        text={name && name.split('')[0]}
+      />
+    );
+  }
+}
+
+function getDefaultAvatar(avatarSize: AvatarSize = AvatarSize.medium) {
```

Comment:
I think we should create a `Anonymous.svg` file in the Avatar's folder, and use it instead of having it inline in this file.
So you'll have something like this:
```tsx
import { ReactElement as AnonymousSVG } from './Anonymous.svg';
...
render() {
 return (
  <AnonymousSVG width={widthBySize} height={heightBySize} />
 )
}
```

Reply:
So you should probably take the biggest SVG, and use it for the file

Reply:
Are you sure this trick works in *ts?

Reply:
Sorry, it's `ReactComponent` and it works in ts :)